src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -202,14 +215,18 @@ def parse_args(argv=None):
     parser.set_defaults(func=handler)
 
     # Conceptually group args for easier visualization
-    add_model_args(parser)
-    add_profile_args(parser)
-    add_endpoint_args(parser)
-    add_dataset_args(parser)
+    _add_model_args(parser)
+    _add_profile_args(parser)
+    _add_endpoint_args(parser)
+    _add_dataset_args(parser)
 
-    args = parser.parse_args(argv)
+    args, extra_args = parser.parse_known_args(argv)
+    if extra_args:
+        # strip off the "--" demarking the pass through arguments
+        extra_args = extra_args[1:]
+        logger.info(f"Additional pass through args: {extra_args}")
```

Comment:
Can be covered in follow-up, but I think this will be fragile. If a user makes a typo or something like:
```bash
genai-perf --typo my_model -- extra_arg_1
```
I think this will capture `--typo` as the first arg here.

There are probably more practical ways to handle this, such as either:
- not having any separator, so all extra_args are valid
- or having a specific named separator arg to denote separation like `--passthrough` or something that can be checked through `args.passthrough`

We can talk more offline or in follow-up though, not a blocker for this PR.